docs/runner/subjects_create_b64.rst

```diff
@@ -0,0 +1,27 @@
+.. _subjects_create_b64_yamlref:
+
+Subjects Create From base64 Story Runner YAML
+.............................................
+
+'subject_label' is not required but if unspecified the created subject will
+not be accessible to later actions in the story.
+
+.. code-block:: yaml
+    
+    ---
+    steps:
+      - step:
+        action: SUBJECTS_CREATE_FROM_B64
+        description: Import a subjects entity.
+        print_response: true
+        subject_label: An imported subject
+      display_name: An imported subject
+      b64data: eyJpZGVudGl0eSI6InN1YmplY3RzLzAwMDAwMDAwLTAwMDAtMDAwMC0wMDAwLTAwMD
```

Comment:
Rather than b64 data can we call this Subject String as we do in the UI?

Reply:
fixed